dbms/src/Common/MemoryTracker.cpp

```diff
@@ -207,6 +210,8 @@ __thread MemoryTracker * current_memory_tracker = nullptr;
 thread_local MemoryTracker * current_memory_tracker = nullptr;
 #endif
 
+std::shared_ptr<MemoryTracker> root_of_mem_trackers_not_managed_by_process_list = MemoryTracker::create();
```

Comment:
change a name like root_of_storage_mem_trackers to make it more specfic?  Just my personal opinion.

Reply:
ok

Reply:
done